### PR TITLE
[EDU-3856] unbind a service

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue:
[EDU-3856](https://aziontech.atlassian.net/browse/EDU-3856)

[EDU-3856]: https://aziontech.atlassian.net/browse/EDU-3856?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+og_image: ''
+permalink: /documentation/products/guides/deploy/unbind-service/
+---
+
+import Button from '~/components/Button.astro'
+
+You can unbind an edge service from an edge node using:
+
+- [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/)
+- [Azion API](https://api.azion.com/#32f34add-9e9f-4b25-94c9-3cee4c266a23)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx:16]:
```suggestion
- [Real-Time Manager (RTM)](#via-rtm)
- [Azion API](#via-api)
```

File: src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+---
+
+## Via RTM 
+
+1. [Access RTM](/en/documentation/products/guides/access-rtm/).
+2. On the upper-left corner of the page, open the **Products menu**, represented by three horizontal lines, and then select **Edge Nodes**.
+3. Select the edge node to which you want to unbind a service.
+4. Go to the **Services** tab.
+6. Hover the service on the list and click the trash can icon.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx:26]:
```suggestion
1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Nodes**.
2. Select the edge node to which you want to unbind a service.
3. Go to the **Services** tab.
4. Hover over the service on the list and click the trash can icon.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx:26]: talvez vale adicionar o passo 5 para explicar o que acontece/fazer logo de clicar o trash icon

File: src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+3. Run the following `DELETE` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/), `:node_id` with the ID of the edge node and `:bind_id` with the ID of the relation between edge node and edge service:
+
+```bash
+curl --location --request DELETE 'https://api.azionapi.net/edge_nodes/:node_id/services/:bind_id' \
+--header 'Accept: application/json; version=3' \
+--header 'Authorization: Token [TOKEN VALUE]'
+```
+
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx:56]:
```suggestion
```

**Review by MarianaAguilera — APPROVED** [review]
algumas sugestões

File: src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx (hunk 2 above)
Comment (gabriel-azion) [inline src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx:26]:
```suggestion
5. Hover the service on the list and click the trash can icon.
```

**Pull request merged**